mariadb_skysql/tests/conftest.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+import pytest
+
+from datadog_checks.dev.http import MockResponse
+
+from .common import MOCKED_MARIADB_SKYSQL_INSTANCE
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+
+@pytest.fixture(scope='session')
+def dd_environment():
+    yield
```

Comment:
The test failure indicates that a configuration is to be yielded by this fixture. For example:

```suggestion
    yield {
        'openmetrics_endpoint': MOCKED_MARIADB_SKYSQL_INSTANCE.get("mariadb_skysql_endpoint"),
        'extra_headers': {'Authorization': data},
    }
```